grisp/default/common/deploy/files/etc/hosts

```diff
@@ -0,0 +1,3 @@
+149.248.205.211 devices.grisp.io
+37.16.30.91 devices.stage.grisp.io
+212.25.1.1 0.europe.pool.ntp.org
```

Comment:
the NTP server IP is subject to change, we should not version it

Reply:
We should rely on dhcp now

Reply:
This was in erl_inetrc, I just moved it there...

Reply:
I'll update the grisp rep and remove this